### DISCUSSION TITLE
* ERROR: unable to parse 'branches'

### DISCUSSION THREAD

**Discussion by C8opmBM** [trapexit/mergerfs#1064] [Q&A]
For general questions and support please use [GitHub Discussions](https://github.com/trapexit/mergerfs/discussions) or [Discord](https://discord.gg/MpAr69V).

Hi guys, 
I have an issue and I'm trying for a day to troubleshoot.
I'm using OMV and I cannot create pool in the GUI, it errors out when applying changes.

In the CLI, if I run the following, everything works perfectly

```
sudo mkdir /mnt/test
sudo mount -t fuse.mergerfs /srv/dev-disk-by-label-Media/Media:/srv/dev-disk-by-label-Media2/Media2 /mnt/test
```

However, with this command in which I also specify the options, it gives this error:

```
sudo mount -t fuse.mergerfs test1:pool1 /mnt/pool -o branches=/srv/dev-disk-by-label-Media/Media:/srv/dev-disk-by-label-Media2/Media2,category.create=mfs,minfreespace=50G,fsname=test1:pool1,defaults,allow_other,cache.files=off,use_ino
* ERROR: unable to parse 'branches' - test1:pool1
* ERROR: mountpoint not set
```

How can I find the naughty option that causes this?
Thank you

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs#synopsis

Using the mergerfs command is the recommended and documented way to use it.

That said it's like any other filesystem to use with mount

```
mount -t fuse.mergerfs -o use_ino,allow_other,... /mnt/a:/mnt/b /mnt/mergerfs
```
